airflow/example_dags/example_task_tags.py

```diff
@@ -0,0 +1,63 @@
+#
```

Comment:
This file is never used by anything -- please don't add new example dags without very good reason as each file in this folder slows down our test suite.

Reply:
Fixed in 8c67348